pandajedi/jedirefine/AtlasAnalTaskRefiner.py

```diff
@@ -131,6 +131,9 @@ def extractCommon(self, jediTaskID, taskParamMap, workQueueMapper, splitRule):
                             break
             except Exception:
                 pass
+        # message driven
+        if "messageDriven" not in taskParamMap:
```

Comment:
Hopefully all analysis tasks will enable the mechanism eventually, but at the beginning we want to see how the system scales gradually. It would be good to introduce a gdpconfig param enabling the stuff for XYZ % of analysis tasks.

Reply:
Currently it is running on one JEDI node so about 11% analysis new tasks have MD. So far it seems okay.
I wonder if randomly enabling MD for XYZ % of task is worth doing, as enabling MD on various number of JEDI nodes does similar job.
I would prefer to define a "MD-priority" of tasks instead of randomly choosing tasks for scaling up/down.

Reply:
The idea to have the gdpconfig param is that DPA or someone centrally controls the load. Otherwise you have to manually tweak JEDI configurations. IMO, all nods must run with the same configuration. For example, currently MD  tasks will disappear if the MD-enabled node goes down. 

Concerning scaling up/down based on MD priority, how does it work if all analysis tasks have the same MD priority?

Reply:
Added USER_TASKS_MESSAGE_DRIVEN_PERCENT in code and gdpconfig